api/src/database/users/users.service.ts

```diff
@@ -27,4 +27,11 @@ export class UsersService {
     return await this.repo.findAll();
   }
 
+  async findOrCreate(number: string): Promise<[ User, boolean ]> {
+    const user = await this.repo.findByPhNo(number);
+    if (user != null)
+      return [ user, false ];
```

Comment:
should be true even if only phno exists and nothing else does exist.

Reply:
Okay